src/plugins/RedPacket/types.ts

```diff
@@ -176,4 +94,65 @@ export namespace History {
         _token_addr: string
         _total_tokens: string
     }
+
+    export interface CreateOutputLog {
+        $name: string
+        total: string
+        id: string
+        creator: string
+        creation_time: string
+        token_address: string
+    }
+
+    export interface ClaimInputLog {
+        $name: string
+        id: string
+        password: string
+        _recipient: string
+        validation: string
+    }
+
+    export interface ClaimOutputLog {
+        $name: string
+        id: string
+        claimer: string
+        claimed_value: string
+        token_address: string
+    }
+
+    export interface RefundInputLog {
+        $name: string
+        id: string
+    }
+
+    export interface RefundOutputLog {
+        $name: string
+        id: string
+        token_address: string
+        remaining_balance: string
+    }
+
+    export interface CreateRedPacketRecord {
+        id: string
+        status: 'refunded' | 'expired' | 'claimed' | 'empty'
```

Comment:
```suggestion
        status: 'refunded' | 'expired' | 'claimed' | 'empty' | null
```

Reply:
I will update when doing the red packet lists in the dashboard.